frontend/src/components/ShootWorkers/WorkerConfiguration.vue

```diff
@@ -13,14 +13,49 @@ SPDX-License-Identifier: Apache-2.0
     width="1250"
     confirm-required
     caption="Configure Workers"
-    disable-confirm-input-focus>
+    disable-confirm-input-focus
+    disable-content-inset
+    max-height="80vh">
+    <template v-slot:tabbar>
+      <v-tabs
+        color="primary"
+        v-model="tab"
+      >
+        <v-tab
+          key="overview"
+          href="#overview"
+        >
+          Overview
+        </v-tab>
+        <v-tab
+          key="yaml"
+          href="#yaml"
+        >
+          Yaml
+        </v-tab>
+      </v-tabs>
+    </template>
     <template v-slot:actionComponent>
-      <manage-workers
-        @valid="onWorkersValid"
-        @additionalZonesNetworkConfiguration="setNetworkConfiguration"
-        ref="manageWorkers"
-        v-on="$manageWorkers.hooks"
-      ></manage-workers>
+      <v-tabs-items v-model="tab">
+        <v-tab-item id="overview">
+          <manage-workers
+            @valid="onWorkersValid"
+            @additionalZonesNetworkConfiguration="setNetworkConfiguration"
+            ref="manageWorkers"
+            v-on="$manageWorkers.hooks"
+          ></manage-workers>
+        </v-tab-item>
+        <v-tab-item id="yaml">
+          <shoot-editor
+            :shoot-item="editorData"
+            :completionPaths="['spec.properties.provider.properties.workers', 'spec.properties.provider.properties.infrastructureConfig']"
+            ref="workerEditor"
+            v-on="$workerEditor.hooks"
+            hide-toolbar
+          >
```

Comment:
on initial load and then switching to the `yaml` tab, the size of the dialog first gets smaller and then large once the data is loaded

Reply:
I tried multiple things to fix this, however the initial problem seems to be that the editor only renders when it is actually visible.
Setting the `eager` flag to the tab item does not help. The editor is mounted, I'm able to set the content beforehand but the rendering does not work. You need to set the content when the tab is active in order to make the editor render properly. I've spend hours now to try to understand this behavior but I'm not getting any further.
Also trying to force to at least keep the hight of the tab makes the animation break.
If someone can provide a solution I would be more than happy. otherwise I would say we leave it for now as it is :shrug:

Reply:
Created issue for this: https://github.com/gardener/dashboard/issues/1332